evap/evaluation/management/commands/import_cms_data.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import urllib.parse
+
+import requests
```

Comment:
how do I fix the mypy error (`Library stubs not installed for "requests")`) ?

Reply:
So far so good. Let's say we don't just ignore it: How do I add a development dependency?